### ISSUE TITLE
optimizer schedule artificially low learning rate

### ISSUE THREAD

**Issue by sachaguer** [sachaguer/jaxili#16]
The scheduler is fixed depending on the number of epochs and steps of the training. The default value of the number of epochs is voluntarily high to stop the training when the validation metrics does not go down anymore.
This has as effect to artifficially lower the learning rate as the schedule will not pick before the end of the training.
